exp/lib/models/content.py

```diff
@@ -0,0 +1,40 @@
+import urllib
+
+from .. import api_utils
+
+class Content(object):
+
+  def __init__(self, document, _isChildrenPopulated=False):
+    self.document = document
+    self._isChildrenPopulated = _isChildrenPopulated
+
+  def get_url(self):
+    subtype = self.document['subtype']
+    if subtype == 'scala:content:file':
+      path = urllib.quote_plus(self.document['path'])
+      return api_utils.generate_url('/api/delivery' + path)
+    elif subtype == 'scala:content:app':
+      path = urllib.quote_plus(self.document['path'] + '/index.html')
+      return api_utils.generate_url('/api/devliery' + path)
+    elif subtype == 'scala:content:url':
+      return self.document['url']
+    raise Exception('Cannot get url for this subtype.')
+
+  def get_variant_url(self, variant_name):
+    subtype = self.document['subtype']
```

Comment:
I think there's potential for bad data here. A try/catch block would be helpful, or at least if/else checks and Exceptions thrown.